db/migrate/20230322201432_add_confirmed_at_to_users.rb

```diff
@@ -0,0 +1,2 @@
+class AddConfirmedAtToUsers < ActiveRecord::Migration[7.0]
+end
```

Comment:
- [ ] I would advise you to delete this migration file because you don't need it. The `devise` gem automatically configures the confirmable module if you activate it in the migration file for `devise` and also add it to the `app/models/users.rb` file in the devise modules. 👍 😎